model/src/test/kotlin/config/ExcludesTest.kt

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.config
+
+import com.here.ort.model.AnalyzerResult
+import com.here.ort.model.Identifier
+import com.here.ort.model.PackageReference
+import com.here.ort.model.Project
+import com.here.ort.model.Scope
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+
+class ExcludesTest : WordSpec() {
+    init {
+        "findProjectExclude" should {
+            val project = Project.EMPTY.copy(definitionFilePath = "path2")
+
+            "return null if there is no matching project exclude" {
+                val excludes = Excludes(
+                        projects = listOf(
+                                ProjectExclude("path1", ProjectExcludeReason.BUILD_TOOL_OF, comment = "comment"),
+                                ProjectExclude("path3", ProjectExcludeReason.BUILD_TOOL_OF, comment = "comment")
+                        )
+                )
+
+                excludes.findProjectExclude(project) shouldBe null
+            }
+
+            "find the correct project exclude" {
+                val projectExclude = ProjectExclude("path2", ProjectExcludeReason.BUILD_TOOL_OF, comment = "comment")
+
+                val excludes = Excludes(
+                        projects = listOf(
+                                ProjectExclude("path1", ProjectExcludeReason.BUILD_TOOL_OF, comment = "comment"),
+                                projectExclude,
+                                ProjectExclude("path3", ProjectExcludeReason.BUILD_TOOL_OF, comment = "comment")
+                        )
+                )
+
+                excludes.findProjectExclude(project) shouldBe projectExclude
+            }
+        }
+
+        "isPackageExcluded" should {
+            val id = Identifier("provider", "namespace", "name", "version")
```

Comment:
Would it make sense to extract those out of the test case in order to increase readability.
Those might be even constants in static context...and also be re-used by other test cases.